### DISCUSSION TITLE
World Food Programme Design System

### DISCUSSION THREAD

**Discussion by Utzel-Butzel** [carbon-design-system/carbon#8915] [Help] [answered]
Hi Carbon Team,
first of all, thank you for the great open source project you've built! We at the [United Nations World Food Programme](https://www.wfp.org/) have our own mostly react-based design system, which is in many parts based on Carbon and is now used across many projects. https://uikit.wfp.org/docs/index.html

Since other UN organizations show interest in using the WFP UI-Kit and we only have very limited resources I'm wondering if it would be appreciated from your side to use the `@carbon` package as an underlying dependency for our own design customizations and components in a similar way you are doing it with [Carbon for IBM.com](https://github.com/carbon-design-system/carbon-for-ibm-dotcom)?

**Comment by emyarod** [comment, marked as answer]
hi @Utzel-Butzel thank you for reaching out! Carbon is licensed under the [Apache 2.0 License](https://github.com/carbon-design-system/carbon/blob/main/LICENSE) which permits usage, modification, and redistribution of Carbon in your projects. hope that answers your question!
